core/modules/filters.js

```diff
@@ -171,15 +171,15 @@ exports.parseFilter = function(filterString) {
 				}
 				if(match[3]) {
 					operation.suffixes = [];
-					 $tw.utils.each(match[3].split(":"),function(subsuffix) {
+						$tw.utils.each(match[3].split(":"),function(subsuffix) {
 						operation.suffixes.push([]);
 						$tw.utils.each(subsuffix.split(","),function(entry) {
 							entry = $tw.utils.trim(entry);
 							if(entry) {
 								operation.suffixes[operation.suffixes.length -1].push(entry);
 							}
 						});
-					 });
+						});
```

Comment:
I think the indent-level here is wrong

Reply:
Thanks @pmario fixed in 4897248809d74e59dc6164c337a6553c8c4607c5